common/lib/xmodule/xmodule/course_module.py

```diff
@@ -958,17 +958,6 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
-    bypass_home = Boolean(
-        display_name=_("Bypass Course Home"),
-        help=_(
-            "Bypass the course home tab when students arrive from the dashboard, "
-            "sending them directly to course content."
-        ),
-        default=False,
-        scope=Scope.settings,
-        deprecated=True
-    )
```

Comment:
@davidjoy, should we do a follow-up to remove this from Mongo too?

Reply:
@ormsbee We don't need to clean up anything in Mongo, right?

Reply:
@stvstnfrd, @Agrendalath: There's no need to clean this up at the Mongo layer. Any extra fields should just be ignored. And in fact, cleaning this up would probably do more harm than good, since the saved structures are assumed to be immutable.